pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -313,6 +313,22 @@ func (p *ConfigurationEditFlags) Apply(
 		}
 	}
 
+	// Deprecated option
+	if cmd.Flags().Changed("min-scale") {
```

Comment:
you could add it to the `if` below with an "or", to avoid duplicated code. Also, I would add an comment for which release we can remove this option.

According to our policy this would be n+2 where n is the first version where this feature was deprecated (so 0.19 iiur)

Reply:
Thank you for the suggestions. Went ahead and make the updates.